src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplate.java

```diff
@@ -502,6 +502,7 @@ InspectImageResponse pullImage(DockerAPI api, TaskListener listener) throws IOEx
                 cmd.exec(new PullImageResultCallback() {
                     @Override
                     public void onNext(PullResponseItem item) {
+                        super.onNext(item);
```

Comment:
This _looks_ like you're fixing an old bug here while doing everything else... Can you explain your reasoning here?
Was this line "always missing but should've been there since the outset" or has this only become an issue after updating to a later version of the docker-java code?

Reply:
It was already missing before the docker-java update. But the underlying code has change a lot in docker-java 3.1.5 and this missing line is now causing a *Could not pull image* error (see https://github.com/docker-java/docker-java/blob/3.1.5/src/main/java/com/github/dockerjava/core/command/PullImageResultCallback.java#L94)